toolbox/bulk_processing/validators.py

```diff
@@ -293,3 +293,30 @@ def fail_validation_if_invalid_uuid(case_id):
         uuid.UUID(case_id, version=4)
     except Exception:
         raise Invalid(f'Cannot look up Case ID {case_id}, it is not a valid UUID')
+
+
+def qid_linked_to_correct_survey_type():
+    def validate(qid, **kwargs):
+        ccs_qids = ['71', '73', '51', '53', '61', '63', '81', '83']
```

Comment:
I don't think this is the full list of CCS QIDs. `'51', '52', '53', '54', '61', '62', '63', '71', '73', '81', '83'` are all the CCS QID's documented. I think a couple of those ended up unused but better to include the full list.

Also this could be a set since it's purpose is checking for inclusion (not that it'll make a big difference for this scale).